### PR TITLE
Feature/add google pay component

### PR DESCRIPTION
**Pull request by Robert-SD**
## Summary
Added google pay component for session flow

### CODE REVIEW

File: android/src/main/kotlin/com/adyen/checkout/flutter/PlatformApi.kt
Comment (Robert-SD) [inline android/src/main/kotlin/com/adyen/checkout/flutter/PlatformApi.kt:1]: Generated code. No review required.

File: ios/Classes/PlatformApi.swift
Comment (Robert-SD) [inline ios/Classes/PlatformApi.swift:1]: Generated code. No review required.

File: lib/src/generated/platform_api.g.dart
Comment (Robert-SD) [inline lib/src/generated/platform_api.g.dart:1]: Generated code. No review required.

**Review by erenbesel — DISMISSED** [review]

File: android/src/main/res/layout/component_bottom_sheet_content.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (OscarSpruit) [inline android/src/main/res/layout/component_bottom_sheet_content.xml:2]: Does this layout scroll correctly with nested scrolling? I can remember issues with bottom sheets and having to use `NestedScrollView` 🤔

File: android/src/main/kotlin/com/adyen/checkout/flutter/components/googlepay/session/GooglePaySessionComponent.kt [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+import com.adyen.checkout.sessions.core.CheckoutSession
+import com.adyen.checkout.sessions.core.SessionSetupResponse
+import com.google.android.material.bottomsheet.BottomSheetDialogFragment
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.launch
+import java.util.Locale
+import java.util.UUID
+
+class GooglePaySessionComponent(
+    private val activity: FragmentActivity,
```
Comment (OscarSpruit) [inline android/src/main/kotlin/com/adyen/checkout/flutter/components/googlepay/session/GooglePaySessionComponent.kt:34]: Same here

File: android/src/main/kotlin/com/adyen/checkout/flutter/components/ComponentLoadingBottomSheet.kt [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+    companion object {
+        private const val TAG = "AdyenComponentLoadingBottomSheet"
+
+        fun <T> show(
+            fragmentManager: FragmentManager,
+            component: T
+        ) where T : ViewableComponent, T : Component {
+            ComponentLoadingBottomSheet(component).apply {
+                isCancelable = false
```
Comment (OscarSpruit) [inline android/src/main/kotlin/com/adyen/checkout/flutter/components/ComponentLoadingBottomSheet.kt:37]: I think it's better to set this in `onCreateView`. Just to make sure it's always set

**Review by OscarSpruit — APPROVED** [review]

**Pull request merged**